src/request.js

```diff
@@ -136,4 +136,19 @@ module.exports = class Request extends EventEmitter {
     }
     return null;
   }
+
+  // Temporarily suspends the flow of data from the database.
+  // No more 'row' events will be emitted until resume() is called.
+  pause() {
```

Comment:
No. From the API point of view, cancel() should be a member of the Request class. It's the request that is beeing canceled, not the connection.